addons/website_sale_delivery/tests/test_ui.py

```diff
@@ -7,6 +7,9 @@
 class TestUi(odoo.tests.HttpCase):
 
     def test_01_free_delivery_when_exceed_threshold(self):
+        # This requires redirect views which are in the data of payment_custom module
+        if not self.env.ref("payment.payment_provider_transfer").redirect_form_view_id:
```

Comment:
Rely on the `state` field of the `ir.module.module` record rather than assessing the existence of the base `payment_custom`'s record's view.

Reply:
No, because the view itself has nothing to do with the problem. If Wire Transfer was implemented to work without that view, or if you changed your check to verify that `custom_mode` is set, it wouldn't change anything.